src/setup.erl

```diff
@@ -134,6 +140,10 @@ enable_cluster_int(Options, no) ->
             config:set("httpd", "bind_address", binary_to_list(NewBindAddress))
     end,
 
+    NodeCount = couch_util:get_value(node_count, Options),
+    ok = require_clustersize(NodeCount),
+    config:set_integer("cluster", "n", NodeCount),
```

Comment:
`N` should not be set to the number of nodes in the cluster; those parameters are largely orthogonal.